projects/step-frontend/src/lib/modules/timeseries/components/chart-dashlet/chart-dashlet.component.html

```diff
@@ -28,7 +28,10 @@
       <step-icon name="x"></step-icon>
     </button>
   </div>
-  @if (!editMode && !((item.readonlyGrouping || item.inheritGlobalGrouping) && item.readonlyAggregate)) {
+  @if (
+    !editMode &&
+    !((item.readonlyGrouping || item.inheritGlobalGrouping) && item.readonlyAggregate && selectedAggregate !== 'RATE')
```

Comment:
```suggestion
    !editMode && (!item.readonlyGrouping && !item.inheritGlobalGrouping || !item.readonlyAggregate || selectedAggregate === 'RATE')
```
I prefer this as it removes one bracket and makes the expression easier to read